### ISSUE TITLE
podman.io only gives the link to the `amd64` version of CLI instaler to `arm64` mac users

### ISSUE THREAD

**Issue by m1kola** [containers/podman.io#275]
### Issue Description

I noticed that [podman.io](https://podman.io/) only has a link to download `amd64` version of CLI installer for mac despite the `arm64` version being available on GitHub ([example](https://github.com/containers/podman/releases/tag/v4.9.3)).

This leads to issues like https://github.com/containers/podman/issues/18512.

### Steps to reproduce the issue

Steps to reproduce the issue
1. On a ARM based Mac go to https://podman.io/
2. Click "Download" and "Podman CLI for macOS" 
3. Install podman
4. Run `podman machine init`
5. Run `podman machine start`

### Describe the results you received

```
Error: Starting machine "podman-machine-default" Waiting for VM ... Error: qemu exited unexpectedly with exit code -1, stderr: qemu-system-x86_64: Unknown Error
```
Due to incorrect arch being installed

### Describe the results you expected

Either `amd64` or universal version of podman CLI is installed and there is no error due to arch when trying to start a VM.

### podman info output

```yaml
N/A - website issue
```

### Podman in a container

No

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

https://podman.io/

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Comment by TomSweeneyRedHat** [comment]
@lsm5 or @baude can either of you fill in the details on the page?  This is the page that will need tweaks: https://github.com/containers/podman.io/blob/main/docs/installation.md

**Comment by benoitf** [comment]
the problem is on the front page

![image](https://github.com/containers/podman.io/assets/436777/1360eff6-f7a5-4885-84f8-b1ce22bb713b)

'Podman CLI for mac redirect to amd64'

**Comment by m1kola** [comment]
@TomSweeneyRedHat the installation page has info about GitHub release page and, I think, it is ok. I'm more concerned about the front page.

**Comment by benoitf** [comment]
https://github.com/containers/podman.io/blob/7f0d60aac3023a545d707426e78901573c98208d/src/components/layout/HeroHeader/index.tsx#L20-L24

the problem is that userAgent returns 'intel' even if you're on silicon/arm
```
['mozilla/5.0', '(macintosh;', 'intel', 'mac', 'os', 'x', '10_15_7)', 'applewebkit/537.36', '(khtml,', 'like', 'gecko)', 'chrome/120.0.0.0', 'safari/537.36']
```

**Comment by benoitf** [comment]
`navigator.userAgentData.getHighEntropyValues(['architecture'])` would work but it's not supported on all browsers

so maybe it would be better to have an universal installer for podman

**Comment by TomSweeneyRedHat** [comment]
@m1kola I missed that, thanks for the clarification!

**Comment by m1kola** [comment]
I believe there is currently no universal installer being built for podman CLI.

I think it is fine to have two links links: no need to detect arch automatically in the browser. It is pretty common experience to select an appropriate arch. Just having it more explicit should be enough.

**Comment by TomSweeneyRedHat** [comment]
Fixed with #289
